src/posts/announce-2.3.0.Final.adoc

```diff
@@ -0,0 +1,185 @@
+---
+title: Announcing Thorntail 2.3.0.Final
+publishDate: 2019-1-17
+modifyDate: 2019-1-17
+author: The Thorntail Team
+layout: post.jade
+---
+
+== Thorntail 2.3.0.Final
+
+TODO
+
+=== What's New?
+
+* Upgrade to WildFly 15.0.1
+* New JSONB fraction
```

Comment:
I would probably replace the new fraction mentions with a general "Update to Java EE 8" type thing

Reply:
I've collapsed JSONB and EE-Security content under a single section, Update to Java EE 8